tests/test_torch_tf_conversion.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
```

Comment:
@thomwolf, @LysandreJik I re-wrote the top_k_top_p_filtering function in tensorflow and since it's quite an important function I feel like we should at least test that TF and PT produce the same results. Not quite sure about the testing format. This file is a version for how I would test it. What do you think @thomwolf , @LysandreJik ?

Reply:
Sure, testing that they both output the same results is cool. Wouldn't we want to test that they output a specific output that we know for sure to be the correct top_p/top_k, to test that they both actually behave as we want? Would seem more robust to me.

Reply:
changed the tests to hardcoded individual integration tests!